ed25519-program/src/lib.rs

```diff
@@ -18,6 +18,11 @@ pub const SIGNATURE_OFFSETS_SERIALIZED_SIZE: usize = 14;
 pub const SIGNATURE_OFFSETS_START: usize = 2;
 pub const DATA_START: usize = SIGNATURE_OFFSETS_SERIALIZED_SIZE + SIGNATURE_OFFSETS_START;
 
+/// Copy from agave-feature-set
```

Comment:
Yeah this is exactly what I was trying to avoid in https://github.com/anza-xyz/solana-sdk/pull/85. By doing this we are coupling sdk crate versions to runtime releases in a way that will be really tricky to track, especially for rekeys.

Reply:
Yep that's totally fair!